server.go

```diff
@@ -405,3 +433,20 @@ func validateAuth(auth string) bool {
 
 	return true
 }
+
+// This creates a Stripe error to return in case of API errors.
+func createStripeError(errorType string, errorMessage string) *ResponseError {
+	stripeError := &ResponseError{
+		ErrorInfo: ErrorInfo{
+			Message: errorMessage,
+			Type:    errorType,
+		},
+	}
+
+	return stripeError
```

Comment:
Can we do away with assigning `stripeError` here and just return `&ResponseError` directly?

Reply:
fixed